draft-ietf-teas-yang-path-computation.txt

```diff
@@ -0,0 +1,4704 @@
+       3.3.1.  Temporary reporting of the computed path state  . . .  28
+   4.  Path computation and optimization for multiple paths  . . . .  30
+   5.  YANG data model for requesting Path Computation . . . . . . .  31
+     5.1.  Synchronization of multiple path computation requests . .  32
+     5.2.  Returned metric values  . . . . . . . . . . . . . . . . .  34
+
+
+
+Busi, et al.              Expires 28 July 2022                  [Page 2]
+
+Internet-Draft          Yang for Path Computation           January 2022
+
+
+     5.3.  Multiple Paths Requests for the same TE tunnel  . . . . .  35
+       5.3.1.  Tunnel attributes specified by value  . . . . . . . .  37
+       5.3.2.  Tunnel attributes specified by reference  . . . . . .  38
+     5.4.  Multi-Layer Path Computation  . . . . . . . . . . . . . .  40
+   6.  YANG data model for TE path computation . . . . . . . . . . .  41
+     6.1.  Tree diagram  . . . . . . . . . . . . . . . . . . . . . .  41
+     6.2.  YANG module . . . . . . . . . . . . . . . . . . . . . . .  52
+   7.  Security Considerations . . . . . . . . . . . . . . . . . . .  72
+   8.  IANA Considerations . . . . . . . . . . . . . . . . . . . . .  73
+   9.  References  . . . . . . . . . . . . . . . . . . . . . . . . .  73
+     9.1.  Normative References  . . . . . . . . . . . . . . . . . .  73
+     9.2.  Informative References  . . . . . . . . . . . . . . . . .  75
+   Appendix A.  Examples . . . . . . . . . . . . . . . . . . . . . .  76
+     A.1.  Basic Path Computation  . . . . . . . . . . . . . . . . .  77
+     A.2.  Path Computation with transient state . . . . . . . . . .  77
+     A.3.  Path Computation with Global Path Constraint  . . . . . .  78
+     A.4.  Path Computation with Per-tunnel Path Constraint  . . . .  79
+     A.5.  Path Computation result . . . . . . . . . . . . . . . . .  80
+   Acknowledgments . . . . . . . . . . . . . . . . . . . . . . . . .  81
+   Contributors  . . . . . . . . . . . . . . . . . . . . . . . . . .  82
+   Authors' Addresses  . . . . . . . . . . . . . . . . . . . . . . .  84
+
+1.  Introduction
+
+   There are scenarios, typically in a hierarchical Software-Defined
+   Networking (SDN) context, where the topology information provided by
+   a Traffic Engineering (TE) network provider may be insufficient for
+   its client to perform end-to-end path computation.  In these cases
+   the client would need to request the provider to calculate some
+   (partial) feasible paths, complementing its topology knowledge, to
+   make end-to- end path computation feasible.
+
+   These types of scenarios can be applied to different interfaces in
+   different reference architectures:
+
+   *  Application-Based Network Operations (ABNO) control interface
+      [RFC7491], in which an Application Service Coordinator can request
+      ABNO controller to take in charge path calculation (see Figure 1
+      in [RFC7491]).
```

Comment:
Editorial:
```suggestion
     in which an Application Service Coordinator can request the
      ABNO controller to perform path calculation  and return the
      path to the Application Service Coordinator (see Figure 1
      in [RFC7491]).
```

Reply:
Fixed in https://github.com/rvilalta/ietf-te-path-computation/pull/96/commits/fc6c5456c7cfaf1ec3ffa97477f75847d2fd428f

Co-Authored-By: sergio belotti <sergio.belotti@nokia.com>